services/spar/src/Spar/App.hs

```diff
@@ -317,6 +318,12 @@ verdictHandlerResult bindCky = catchVerdictErrors . \case
       Just cky -> pure $ VerifyHandlerGranted cky uid
       Nothing -> throwSpar $ SparBrigError "sso-login failed (race condition?)"
 
+logVerdictHandlerResult :: VerdictHandlerResult -> Spar VerdictHandlerResult
```

Comment:
Returning `VerdictHandlerResult` implies that this edits it in some way; please return `()` instead even if it means you can't 'chain' the operator; there are ways around that at the call-site.